connector_infor_account_move/messages/move.xml.tmpl

```diff
@@ -0,0 +1,105 @@
+              {% set field_value = base_object.mapped(custom_field.value)[0] %}
+              {# TODO add custom_field.default if field_value is empty #}
+            {% endif %}
+            {% if custom_field.field_type == 'dimensioncode' %}
+              <DimensionCode sequence="{{ index }}" listID="{{ custom_field.name|default('', true) }}">{{ field_value|default('', true) }}</DimensionCode>
+            {% elif custom_field.field_type == 'property' %}
+            <Property sequence="{{ index }}">
+              <NameValue name="{{ custom_field.name|default('', true) }}" type="String" unitCode="EA">{{ field_value|default('', true) }}</NameValue>
+            </Property>
+            {% endif %}
+{%- endmacro %}
+    <Sender>
+      <LogicalID schemeVersionID="6.02.01.0188">lid://infor.sunsystems.1</LogicalID>
+      <ComponentID>{{ COMPONENT_ID|default('', true) }}</ComponentID>
+      <ConfirmationCode>Always</ConfirmationCode>
+      <AuthorizationID />
+    </Sender>
+    <CreationDateTime>{{ CREATE_DATE }}</CreationDateTime>
+    <BODID>Infor-nid:Infor:{{ BUSINESS_UNIT|default('', true) }}::{{ INVOICE_ID|default('', true) }}:0?SourceSystemJournalEntry&amp;verb={{ VERB }}</BODID>
+  </ApplicationArea>
+  <DataArea>
+    <{{VERB}}>
+      <TenantID>{{ TENANT_ID|default('', true) }}</TenantID>
+      <AccountingEntityID>{{ ACCOUNTING_ENTITY_ID|default('', true) }}</AccountingEntityID>
+      <ActionCriteria>
+        <ActionExpression actionCode="Add" />
+      </ActionCriteria>
+    </{{VERB}}>
+    <SourceSystemJournalEntry>
+      <JournalEntryHeader>
+        <JournalID accountingEntity="{{ BUSINESS_UNIT }}" variationID="{{ VARIATION_ID }}">{{ INVOICE_ID|default('', true) }}</JournalID>
+        <DisplayID>{{ INVOICE_ID|default('', true) }}</DisplayID>
+        <BaseJournalEntryHeader>
+          <LedgerID>A</LedgerID>
+          <LedgerType>Actual</LedgerType>
+          <AccountingJournalReference>
+            <ID>{{ JOURNAL_CODE|default('', true) }}</ID>
+          </AccountingJournalReference>
+          <LedgerSetupReference>
+            <DocumentID>
+              <ID>A</ID>
+            </DocumentID>
+            <BookInstanceID>A</BookInstanceID>
+          </LedgerSetupReference>
+          <JournalStatus>
+            <Code>Final</Code>
+          </JournalStatus>
+        </BaseJournalEntryHeader>
+      </JournalEntryHeader>
+      {% for line in JOURNAL_LINES %}
+      <JournalEntryLine sequence="{{ loop.index }}">
+        <Amount currencyID="{{ line.currency_id.name or COMPANY_CURRENCY|default('', true) }}">{{ "%.5f"|format(amount_currency or line.credit or line.debit|default(0, true)) }}</Amount>
+        <FunctionalAmount currencyID="{{ COMPANY_CURRENCY|default('', true) }}">{{ "%.5f"|format(line.credit or line.debit|default(0, true)) }}</FunctionalAmount>
+        <ReportingCurrencyAmount currencyID="{{ REPORTING_CURRENCY|default('', true) }}">{{ REPORTING_AMOUNT}}</ReportingCurrencyAmount>
```

Comment:
@TDu With the following example:
![screenshot from 2018-07-18 16-13-55](https://user-images.githubusercontent.com/939952/42908265-15626fbc-8aa6-11e8-9b0e-eb5b339ca602.png)

I get:
![screenshot from 2018-07-18 16-14-17](https://user-images.githubusercontent.com/939952/42908128-adf406f6-8aa5-11e8-8a52-caf0a1d2d40c.png)

I am expecting:
```
<Amount currencyID="EUR">100.00000</Amount>
<FunctionalAmount currencyID="USD">83.33000</FunctionalAmount>
```